share/docker.sh

```diff
@@ -1018,10 +1104,4 @@ docker_compose_run()
     # Wait on our backgrounded process and propogate any failure from it.
     edebug "Waiting for docker-compose to complete"
     wait "${pid}"
-
-    # Optionally capture all container logfiles
-    if [[ -n "${logfile}" ]]; then
-        edebug "Capturing docker-compose logfile"
-        docker-compose --file "${file}" logs > "${logfile}"
-    fi
```

Comment:
By moving this into a trap, we are ensuring we capture logs even on failure. Before we were only getting logs on **SUCCESS**